README.md

```diff
@@ -1 +1,88 @@
-# BlogApp
\ No newline at end of file
+![Microverse](https://img.shields.io/badge/-Microverse-%20%236f23ff?style=for-the-badge) 
+
+![Ruby](https://img.shields.io/badge/ruby-%23CC342D.svg?style=for-the-badge&logo=ruby&logoColor=white) 
+![Rails](https://camo.githubusercontent.com/2f7aa65a22dedd331620c426fd36d88c01600b2c8ed750c41cc72b4d86364937/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f7261696c732d2532334343303030302e7376673f7374796c653d666f722d7468652d6261646765266c6f676f3d727562792d6f6e2d7261696c73266c6f676f436f6c6f723d7768697465)
+
+# Rails Blog App 
+
+> The Blog app is ruby on rails website that will show the list of post and users to interact by adding comments and liking posts. 
+
+
+## Built With :hammer_and_wrench: 
+
+- **Ruby:** Popular programming language that’s used for lots of things, from web app development to data analysis.
+
+- **Rails:**  Full-stack framework. It ships with all the tools needed to build amazing web apps on both the front and back end.
+
+- **GitHub flow:**  Correct use of Branches for deployment and features development.
+
+- **Linters Check (Rubocop):** Local and Pull Request check for errors, bugs and stylistic errors in code.
+
+- **RSpec:** Unit test framework for the Ruby programming language.
+
+- **PostgreSQL:** powerful, open source object-relational database system with over 35 years of active development that has earned it a strong reputation for reliability, feature robustness, and performance.
+
+## Entity Relationship Diagram 
+
+![](./blog_app_erd.png)
+
+
+## Getting Started ➡️
+
+To get a local copy up and running follow these simple steps.:arrow_down:
+
+### Prerequisites ✅
+- A code editor like Visual Studio Code with Git and Ruby.
+- You can check if Git is installed by running the following command in the terminal: 
+
+```
+$ git --version
+```
+
+Likewise for Ruby installation.
+
+```--sh 
+$ ruby --version
+``` 
+
+### Setup
+
+1. Clone the GitHub repository using the  this link 
+```--sh
+  $ git clone https://github.com/aron-helu/BlogApp.git
+```
+2. Checkout to the `dev` branch
+3. Run `$ bundle install` in your terminal to install all gems required to run the app.
+4. Go to your terminal and run the app with the following command:
+```
+$ rails server
+```
+- **Navigate:** to http://localhost:3000/ 
+
+
+## Authors :writing_hand:
+
+:man_technologist:  **Aron Abraham** :writing_hand: 
+
+- GitHub: [@Aaron](https://github.com/aron-helu)
+- LinkedIn: [@Aaron](https://www.linkedin.com/in/aron-abraham-90a4321b0/)  
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
[OPTIONAL] To make your project professional, kindly add MIT license to your project.

Reply:
@Rustamxon7 Hello! Can you please check readme file? Already have implemented it.

Reply:
Hello, Could you show the license, please? 😊 In your readme file, it supposes to work correctly. However, If I try to open it, it shows me a broken link.